recipe/meta.yaml

```diff
@@ -1,16 +1,16 @@
 {% set name = "ruamel.yaml" %}
-{% set version = "0.16.12" %}
+{% set version = "0.17.21" %}
```

Comment:
The package has many restrictions downstream packages, e.g, <0.15.0, <0.16.0, <0.17.0, or <0.18.0 upper bounds https://pypi.org/project/ruamel.yaml/. I guess that's why `conda` uses `ruamel_yaml` (a patched copy of ruamel.yaml)

Reply:
The way I read this, we are still okay. The current version is `0.16.12`, so packages that don't pin the version already don't use a "pre-0.16" version. `0.17.21` is the last version to use old functionality, so we need to be careful about upgrading to version `0.18` when it comes out.